docs/java/guides/cf-xsuaa.md

```diff
@@ -0,0 +1,229 @@
+---
+id: cf-xsuaa
+title: SAP Cloud Foundry
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Cloud Foundry XSUAA Usage
+custom_edit_url: https://github.com
+description: Configure you Cloud Foundry CLI and bind it to SAP Cloud Platform
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+- xsuaa
+- destination
+- connectivity
+image:
+---
+
+## XSUAA Usage on Cloud Foundry
+
+When implementing and deploying an application it quickly becomes important to understand how authentication and authorizations works on SCP Cloud Foundry.
+In our tutorials and documentation pages we recommend using "approuter" as proxy service to handle authentication management to your implemented application.
+The following steps will show what happens behind the scenes. The requests can be manually reproduced by a REST client of your choice, e.g. Postman or Insomnia.
+
+The [official UAA documentation](https://docs.cloudfoundry.org/api/uaa/) is highly recommended if there are questions.
+
+**Please note:** The following paragraphs only describe a subset of features of the XSUAA Service on Cloud Foundry.
+The Cloud SDK is not participating in the active development of this software and cannot give in-depth support.
+Some information may already be outdated.
+
+
+These are use cases described below:
```

Comment:
```suggestion
The Cloud SDK handles the following use-cases:
```

Reply:
The Cloud SDK does not handle all those cases:

- [ ] User Login: `Authorization Code Grant` (approuter does it)
- [x] SCP Service Usage on behalf of a User: `JWT Bearer Token Grant`
- [x] SCP Service Usage on behalf of a service: `Client Credentials Grant`
- [ ] Resolve User Access Token: `Refresh Token Grant` (approuter does it)